README_orig.md

```diff
@@ -0,0 +1,131 @@
+HClib
```

Comment:
@srirajpaul is there any reason to keep this file around? I'm okay with deleting it, we'll always have this in the git history if we need to change anything.

Reply:
@agrippa Just kept for history. Since it is in git history we can delete this file.